xpublish/routers/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .base import base_router
 from .common import common_router, dataset_collection_router
 from .zarr import zarr_router
+from .xyz import xyz_router
```

Comment:
We should wrap this import in a `try... except` block so it fails silently if the libraries required for this router are not installed.